scripts/tag-extension.js

```diff
@@ -0,0 +1,30 @@
+const fs = require("fs");
+const path = require("path");
+const { spawnSync } = require("child_process");
+
+const pkgJson = require(path.join(
+  __dirname,
+  "..",
+  "apps",
+  "extension",
+  "client",
+  "package.json"
+));
+
+const tag = `${pkgJson.name}@${pkgJson.version}`;
+
+const { status, stdout } = spawnSync("git", [
+  "ls-remote",
+  "https://github.com/statelyai/xstate-tools.git",
```

Comment:
note that this relies on this repo being renamed as planned

Reply:
Can we keep it the current name now just for testing? That way I can rename it all in one go

Reply:
done